packages/components/organisms/f-header/src/components/_tests/Logo.test.js

```diff
@@ -4,7 +4,7 @@ import Logo from '../Logo.vue';
 describe('Logo', () => {
     it('should be defined', () => {
         const propsData = {
-            theme: 'je',
+            theme: 'jet',
```

Comment:
Do we still need this prop?

Reply:
We need this prop for setting the right logo, but actually I noticed that some tests should fail with changing `je` to `jet` but they don't :) I will dig into this :+1:

Reply:
Ah right, cool. OK, I'll keep an eye on any changes to the PR.

Reply:
Ok, I think now the changes make more sense :)